src/components/v5/common/Table/utils.tsx

```diff
@@ -31,19 +30,13 @@ export const makeMenuColumn = <T,>(
       const props = getMenuProps?.(row);
 
       return props ? (
-        <div
-          className={clsx({
-            'absolute top-1/2 right-4 -translate-y-1/2': verticalOnMobile,
-          })}
-        >
-          <MeatBallMenu
-            {...props}
-            contentWrapperClassName={clsx(
-              props.contentWrapperClassName,
-              '!z-[60]',
-            )}
-          />
-        </div>
+        <MeatBallMenu
+          {...props}
+          contentWrapperClassName={clsx(
+            props.contentWrapperClassName,
+            '!z-[65] md:!z-[60]',
```

Comment:
Setting different z-indexes based on the screen size makes it look like we've got a bigger problem presumably with another component depending on this? Is it possible to come up with a single value?

Reply:
It's not needed here anymore. I forgot to remove it.